.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,57 @@
+---
+name: Feature Request
+description: Suggest an idea for this project
+labels: [feature-request]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for using our library and trying to make it better!
```

Comment:
I won't go that far honestly :sweat_smile:

Reply:
Sure